docs/op-stack/src/docs/build/getting-started.md

```diff
@@ -150,11 +150,11 @@ Sequencer: 0x0324a4c8c1955cb8364e8f07558238b3d2aa5f55
 Private Key: fba31658f320bb8ce1ce39fab3c7c2acea6b4dd69cc8483fd85388a461d8426b
 ```
 
-Save these accounts and their respective private keys somewhere, you’ll need them later. Fund the `Admin` address with a small amount of Goerli ETH as we’ll use that account to deploy our smart contracts. You’ll also need to fund the `Proposer` and `Batcher` address — note that the `Batcher` burns through the most ETH because it publishes transaction data to L1.
+Save these accounts and their respective private keys somewhere, you’ll need them later. Fund the `Admin` address with a good amount of Goerli ETH as we’ll use that account to deploy our smart contracts. You’ll also need to fund the `Proposer` and `Batcher` address — note that the `Batcher` burns through the most ETH because it publishes transaction data to L1.
```

Comment:
Might be worth giving some sort of basic formula that accounts for the current gas price. Last night it took 5 ETH with a gas price of 150.